src/commands/deployImage/imageSource/containerRegistry/acr/createAcr/ICreateAcrContext.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { KnownSkuName, Registry } from "@azure/arm-containerregistry";
+import { IResourceGroupWizardContext } from "@microsoft/vscode-azext-azureutils";
+import { ExecuteActivityContext } from "@microsoft/vscode-azext-utils";
+
+export interface ICreateAcrContext extends IResourceGroupWizardContext, ExecuteActivityContext {
+    registryName?: string;
+    sku?: KnownSkuName;
+    registry?: Registry;
+    newResourceGroupName?: string;
+}
```

Comment:
Nit:  I think `IResourceGroupWizardContext` already provides `newResourceGroupName`, so you can probably omit that.  I also slightly prefer `registryName`  to be named `newRegistryName` to match our typical naming convention.